core/block_svg.js

```diff
@@ -262,6 +262,7 @@ Blockly.BlockSvg.prototype.getIcons = function() {
 /**
  * Set parent of this block to be a new block or null.
  * @param {Blockly.BlockSvg} newParent New parent block.
+ * @override
```

Comment:
Challenge here is the superclass (Block) and subclass (BlockSvg) define different parameter types for newParent, one is Block the other is BlockSvg and that conflicts. 

more details about override field:
https://github.com/google/closure-compiler/wiki/@abstract-classes-and-methods